### PR TITLE
gamescope: 3.14.2 -> 3.14.3

### PR DESCRIPTION
**Pull request by K900**
## Description of changes

This has the native Wayland backend with HDR support.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by returntoreality** [comment]
Result of `nixpkgs-review pr 303785` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>bottles</li>
    <li>bottles-unwrapped</li>
    <li>gamescope</li>
    <li>gamescope-wsi</li>
  </ul>
</details>
Also tested gamescope and it works

**Comment by VeilSilence** [comment]
Is `capSysNice = true;` working for you?
Launching from terminal `gamescope glxgears` for example and it works correctly, but when i'm trying to launch any game on steam with this command line `gamescope -w 2560 -h 1440  -f -r 240 --force-grab-cursor  gamemoderun  --mangoapp %command%`  i got this error: `failed to inherit capabilities: Operation not permittedUploaded AppInterfaceStats to Steam`
Also `--mangoapp` not showing nvidia related sensors. Normal `mangohud` working.

**Comment by K900** [comment]
This is a known issue that's unrelated to this PR.

**Comment by diniamo** [comment]
Just to clarify, are both of the issues unrelated to the PR?

**Comment by K900** [comment]
Correct.

File: pkgs/by-name/ga/gamescope/package.nix [hunk 1]
```diff
@@ -99,7 +101,7 @@ stdenv.mkDerivation (finalAttrs: {
     glm
   ] ++ lib.optionals enableWsi [
     vulkan-headers
-  ] ++ lib.optionals enableExecutable [
+  ] ++ lib.optionals enableExecutable (wlroots.buildInputs ++ [  # gamescope uses a custom wlroots branch
```
Comment (PedroHLC) [inline pkgs/by-name/ga/gamescope/package.nix:104]: Interesting and simple solution

**Review by PedroHLC — APPROVED** [review]

**Pull request merged**